### ISSUE TITLE
Support images in application

### ISSUE THREAD

**Issue by MarinTrpenovski** [MarinTrpenovski/Spring-Rest#120]
Need to create REST endpoints that will allow users to load/save images (logos) for each of the entities like faculty, university, student....

The idea is that when uploading the image, the content of the image (bytes) needs to be stored in a server (computer in your case), and in each of the entities table in the Database needs to create a new column img which will accept string values. Here we will save the path where the image is saved on the server (local machine).

On load entity from DB, need to get in the column as well in the JSON response. img column will represent the location where the image is saved. 

Additionally need to create a REST endpoint that will load all images (paths to the location where they are stored) for specific entities. The idea is to have for each of the entities a separate gallery of images so users in the UI are able to select which image to use for the entity and saved in img column in the entity.

These REST endpoints need to be protected as well and only authorized users to be able to access them. 

Use this link as reference https://www.codejava.net/frameworks/spring-boot/spring-boot-file-upload-tutorial

**Comment by AngelaM123** [comment]
Created REST endpoints that allow users to load/save images made changes in database as well

**Comment by MarinTrpenovski** [comment]
@AngelaM123 Code looks good, I notice changes in Model classes so it means that DB is modified as well. 

But I think still need to create 2 rest endpoints in each of the entities:

1. Is to load all images defined per entity..... Idea is to have folder as you have for entity /faculty and load all images from that folder when user trigger appropriate service.
2.  Create DELETE rest endpoint that will delete images from file system and delete image from entity in DB. This should be applied in each of the entities.

**Comment by AngelaM123** [comment]
Yes the database is already modified , also created the new required rest endpoints in each entity

**Comment by MarinTrpenovski** [comment]
Code looks good @AngelaM123 

Delete image endpoint need to be modified , endpoint should delete specific one image not whole folder to the entity. Example passing image name to the rest endpoint and only that image should be removed from the folder and from DB as well.
